draft-ietf-rats-msg-wrap.md

```diff
@@ -587,6 +587,14 @@ As an example, a `cbor-record` containing an UCCS (Unprotected CWT Claims Sets)
 
 This document introduces a format for holding multiple CMW items in a collection.
 If the collection is not protected from tampering by external security measures (such as object security primitives) or internal mechanisms (such as intra-item binding), an attacker could easily manipulate the collection's contents.
+In particular, when a CMW is used to carry multiple Evidence messages for a composite device or layered attestation, there should be strong binding between the Evidence messages within the collection.
```

Comment:
```suggestion
In particular, when a CMW is used to carry multiple Evidence messages for a composite device or layered attestation, there SHOULD be strong binding between the Evidence messages within the collection.
```

Reply:
As discussed, with the new L590 these "should"s are clearly non-normative.